### PR TITLE
added browser extension link and new menu in receive tab.

### PR DESCRIPTION
**Pull request by rijusougata13**

### CODE REVIEW

**Comment by rijusougata13** [comment]
Also fixed undefined url when click on ios and android. PTAL @zaryab2000

**Comment by zaryab2000** [comment]
Looks good to me @rijusougata13 .

Added @Xand6r as a reviewer for a quick review and then we can merge

File: packages/react-app/src/components/ViewInfoItem.js [hunk 1]
```diff
@@ -13,6 +13,20 @@ function ViewInfoItem() {
   // render
```
Comment (Xand6r) [inline packages/react-app/src/components/ViewInfoItem.js:13]: perhaps we could do :
```
const { REACT_APP_BROWSER_EXTENSION_URL } = process.env;
```
here

File: packages/react-app/src/components/ViewInfoItem.js [hunk 2]
```diff
@@ -13,6 +13,20 @@ function ViewInfoItem() {
...
   return (
     <>
+    <Container >
+      <ChannelInfo>
+        <ChannelTitle>
+          <>
+          <AppLinkText>
+            <AppLink href={`${process.env.REACT_APP_BROWSER_EXTENSION_URL}`} target="_blank" rel="nofollow">
```
Comment (Xand6r) [inline packages/react-app/src/components/ViewInfoItem.js:21]: then :
```
<AppLink href={REACT_APP_BROWSER_EXTENSION_URL} target="_blank" rel="nofollow">
```
here.

**Review by Xand6r — CHANGES_REQUESTED** [review]

**Pull request merged**